iterate_snowflake/README.md

```diff
@@ -8,28 +8,31 @@ First of all, you must connect your Tinybird workspace with your Git repository
 - Create a new git branch 
 - Create a new Snowflake .datasource data file in the branch, as in the example PR. You shouldn't need fixtures as the Data Source will be populated from the connection, but you can add fixtures and/or tests if needed.
   - For information about Snowflake options read the [connector documentation](https://www.tinybird.co/docs/ingest/snowflake)
-- Commit your code and create a new PR/MR from the branch to `main`
-- Wait for CI. You can check in the temporary workspace branch (automatically created and named like `tmp_ci_*`) that everything works as expected.
+- Commit your code and create a new PR/MR from the branch to `main` 
+- Wait for CI to finish.
+- Check that everything is working as expected in the temporary branch, CI will automatically create it and name it like `tmp_ci_{PRid}`).
```

Comment:
Could we be more specific? How do I check that everything is working, do I need to `sync now` the data source?